pandas/tests/indexes/test_multi.py

```diff
@@ -154,7 +154,7 @@ def assert_matching(actual, expected):
             # as much as possible
             self.assertEqual(len(actual), len(expected))
             for act, exp in zip(actual, expected):
-                act = np.asarray(act)
+                act = np.asarray(act, dtype=np.object_)
```

Comment:
Because the expected is changed to `np.object_` and if I don't change `act` also the test fails because the expected and actual are not changed.